lisa/tests/scheduler/load_tracking.py

```diff
@@ -310,7 +315,8 @@ def get_simulated_pelt(self, task, signal_name):
             # executing
             preempted_value=0,
         )
-        df = trace.analysis.load_tracking.df_task_signal(task, signal_name)
+        cpu = self.wlgen_task.children['test']['cpus']
```

Comment:
We could add the following property (taken from the deleted CPUMigrationBase and slightly modified to not fail if a phase is not pinned) to ensure we don't miss any CPU and so that we don't need to repeat that bit if it's needed anywhere else:
```
    @property
    def cpus(self):
        """
        All CPUs used by RTapp workload.
        """
        return set(itertools.chain.from_iterable(
            phase.get('cpus', [])
            for task in self.rtapp_profile.values()
            for phase in task.phases
        ))
```